packages/viewer-demos/src/pages/CustomEnv.tsx

```diff
@@ -55,25 +56,68 @@ function CustomEnv(): ReactElement {
 				style={{ height: panelSize, width: "100%" }}
 				model-url="./common-assets/models/WaterBottle.glb"
 				environment={`./common-assets/environments/${loadedEnv}.env`}
-				create-ground
-				create-skybox
+				create-ground={showGround || undefined}
+				create-skybox={showSkybox || undefined}
+				background-color={backgroundColor}
 			/>
-			<FormControl fullWidth>
-				<Select id="model-select" value={loadedEnv} onChange={handleChange}>
-					{envNames.map((name) => (
-						<MenuItem key={name} value={name}>
-							{name}
-						</MenuItem>
-					))}
-				</Select>
+			<div style={{ display: "flex", flexDirection: "column", padding: 12, alignItems: "stretch", gap: 12 }}>
 				{/* eslint-disable prettier/prettier */}
 				<HTMLSnippet>
 					&lt;product-viewer model-url=&quot;./common-assets/models/WaterBottle.glb&quot;
-					environment=&quot;./common-assets/environments/{loadedEnv}.env&quot; create-ground create-skybox
-					/&gt;
+					environment=&quot;./common-assets/environments/{loadedEnv}.env&quot;{" "}
```

Comment:
The encoded quotes feels a bit awkward/hacky to me, is there any other way we could generate this?

Reply:
Updated this to pass the html snippet in as a string and I think it's a little cleaner.